### PR TITLE
Added Indonesia Translation (with some additions) in some pages.

### PR DESCRIPTION
**Pull request by GNZTMPZ**
~~I'll brief what's changed, or just track the commit to see what's changed. (still wip though, do not merge)~~

What's changed:

- move some files to designated place (it was at the wrong place, oof)
- translate some "scripting/functions" to indonesian
- some minor typofix
- and another small details that i forgot to add here.. just look at the commit history 😅 

If @Aiuraa accept my PR, feel free to merge this :>

### CODE REVIEW

**Review by leonardssh — APPROVED** [review]

**Comment by Aiuraa** [comment]
@leonardssh how can i review changes if you already aprove it :\

He ask me on discord to review the changes......

**Comment by leonardssh** [comment]
My approval doesn't prevent you from reviewing. @Aiuraa

**Comment by GNZTMPZ** [comment]
I still working on translating, i just need some advice though. pls do not merge this, *yet*!

I'll ping a member if it's done. themk :>

**Comment by Aiuraa** [comment]
> My approval doesn't prevent you from reviewing. @Aiuraa

It does @leonardssh, since i don't updated CODEOWNERS, when someone approved the change the option is not available anymore.

File: docs/translations/id/scripting/functions/BanEx.md
Comment (Aiuraa) [inline docs/translations/id/scripting/functions/BanEx.md:66]: > Timer harus digunakan untuk menunda ban.

Gak perlu pakai titik, bisa pakai koma dan tambahkan seperti:
", oleh karena itu timer harus digunakan untuk menunda pelarangan"

File: docs/translations/id/scripting/functions/Kick.md
Comment (Aiuraa) [inline docs/translations/id/scripting/functions/Kick.md:9]: Deskripsi "Kick" bisa diganti dengan "Tendang"

File: docs/translations/id/scripting/functions/Kick.md
Comment (Aiuraa) [inline docs/translations/id/scripting/functions/Kick.md:13]: "di kick" bisa diganti menjadi "ditendang"

File: docs/translations/id/scripting/functions/Kick.md
Comment (Aiuraa) [inline docs/translations/id/scripting/functions/Kick.md:38]: Ini juga diganti jadi "Tendang", tidak perlu menggunakan "kick"

File: docs/translations/id/scripting/functions/Kick.md
Comment (Aiuraa) [inline docs/translations/id/scripting/functions/Kick.md:43]: Komentar ini harusnya bisa diganti menjadi:

"Kemudian, buat sebuah timer untuk menunda fungsi Kick selama sedetik"

File: docs/translations/id/scripting/functions/tolower.md
Comment (Aiuraa) [inline docs/translations/id/scripting/functions/tolower.md]: "Function" dapat diubah menjadi "Fungsi"

File: docs/translations/id/scripting/functions/tolower.md
Comment (Aiuraa) [inline docs/translations/id/scripting/functions/tolower.md:34]: Deskripsinya jangan lupa.

File: docs/translations/id/scripting/functions/toupper.md
Comment (Aiuraa) [inline docs/translations/id/scripting/functions/toupper.md:34]: Deskripsinya jangan lupa

File: docs/translations/id/scripting/functions/strcmp.md
Comment (Aiuraa) [inline docs/translations/id/scripting/functions/strcmp.md:11]: Deskripsinya harusnya sesuai sama yang lain, menggunakan "Fungsi ini", lalu dilanjut dengan "membandingkan kedua string"

**Review by Aiuraa — CHANGES_REQUESTED** [review]
Thanks for your contribution!, there is room for improvement that you can do, and also i saw `# Related Function` is not really been translated to indonesian on some places making it inconsistent.

File: docs/translations/id/scripting/functions/BanEx.md
Comment (GNZTMPZ) [reply docs/translations/id/scripting/functions/BanEx.md]: eh mangap, ini kelupaan 😅

File: docs/translations/id/scripting/functions/tolower.md
Comment (GNZTMPZ) [reply docs/translations/id/scripting/functions/tolower.md]: weird, tp sebenarnya ini udah di fix tauu.. kok masi kek gini?

aneh dah gh 🤔

File: docs/translations/id/scripting/functions/tolower.md
Comment (GNZTMPZ) [reply docs/translations/id/scripting/functions/tolower.md]: sebenarnya dr document awalnya gak ada kak, makanya gak dimasukin

namun, klo mau dimasukin yowes tak masukin 👍

File: docs/translations/id/scripting/functions/strcmp.md
Comment (GNZTMPZ) [reply docs/translations/id/scripting/functions/strcmp.md]: sebenarnya dr document awalnya gak ada kak, makanya gak dimasukin

namun, klo mau dimasukin yowes tak masukin 👍

File: docs/translations/id/scripting/functions/toupper.md
Comment (GNZTMPZ) [reply docs/translations/id/scripting/functions/toupper.md]: sebenarnya dr document awalnya gak ada kak, makanya gak dimasukin

namun, klo mau dimasukin yowes tak masukin 👍

**Comment by SanityZeroPercent** [comment]
Hey @leonardssh, sorry for bothering you.. 

We really want to help translating documentations and maintain them. Anyway, I've added my name to the CODEOWNERS, but github said that we don't have write access. I wonder if me, along with @Aiuraa and @rsetiawan7 could have the write access if that's possible. 

Thanks.

**Review by Aiuraa — APPROVED** [review]
I think it's good and can be safe to merge, thanks for your contributions though!

**Review by rsetiawan7 — APPROVED** [review]
lgtm

**Comment by leonardssh** [comment]
Thanks for your contributions!

**Pull request merged**
